Header header
uint16 width
uint16 height
uint8 channels
uint8[] image_left
uint8[] image_right
# image contient les donnees sous differentes formes selon la compression
uint8 compressionRatio
float64 timestamp
